scripts/mkcloud

```diff
@@ -1404,7 +1416,7 @@ function sanity_checks()
 
 
 ## MAIN ##
-step_aliases="_new_admin _compute _upgrade _testupdate"
+step_aliases="blank_admin crowbar crowbar_admin updated_crowbar_admin updates nodes _upgrade _testupdate"
```

Comment:
> Because it could be useful to call from the CLI.

Then we should maybe adapt `usage()` as well
Maybe like this
``` bash
function usage()
{
    cat <<EOUSAGE
Usage:
$0 <step> [<step>,...]

'step' is one of:
    $step_aliases $allcmds
...
```
or an extra line with just the aliases...